AppInspector.CLI/Writers/DummyWriters.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+using Microsoft.ApplicationInspector.Commands;
+
+namespace Microsoft.ApplicationInspector.CLI
+{
+    public class AnalyzeDummyWriter : CommandResultsWriter
```

Comment:
Why do you need so many of these subclasses? Nothing about them is unique.

Reply:
The json writers in most cases are the same but the text writers are distinct by type with different checks and formatting in some cases like the analyze writer including its html writer.  While I could consolidate a few of the json writers, it has some consistency to hold one per format and command type for identifying where that work is done and future distinctions for type.  I'm open to considering reducing where possible -let me chew on.

Reply:
You could remove the boilerplate code here (and keep the classes if you want them) by changing the functions in the base CommandResultsWriter class to just throw NotImplemented (which I assume is the base case) and then you don't need all these overrides - alternately have a "DummyCommandResultsWriter" that has the notimplementedexceptions and then make subclasses of that.